kingfisher_scrapy/spiders/costa_rica_poder_judicial_records.py

```diff
@@ -29,4 +36,10 @@ def parse_list(self, response):
         data = response.json()
         for resource in data['result']['resources']:
             if resource['format'].upper() == 'JSON':
+                if self.from_date and self.until_date:
+                    # URL looks like https://pjcrdatosabiertos.blob.core.windows.net/datosabiertos/OpenContracting/2021
+                    # .json
```

Comment:
Avoid breaking the URL across lines

```suggestion
                    # URL looks like:
                    # https://pjcrdatosabiertos.blob.core.windows.net/datosabiertos/OpenContracting/2021.json
```